sklearn/utils/validation.py

```diff
@@ -155,6 +156,14 @@ def _shape_repr(shape):
     return "(%s)" % joined
 
 
+def check_memory(memory):
```

Comment:
You need to test this functions in `test_validation.py`. Create the `DummyMemory` and `WrongDummyMemory` classes there.